README.dockerhub.md

```diff
@@ -1,3 +1,5 @@
+[//]: # (!!!!IF YOU MAKE CHANGES TO THIS FILE, REMEMBER TO UPDATE DOCKER HUB!!!!)
```

Comment:
TIL a markdown comment looks hella weird https://stackoverflow.com/a/20885980/18424743

Reply:
And how do we update docker hub? Is it a manual process, are the credentials available in 1
Password

Reply:
Yes it's a manual process. Have to log in as a shared account from 1Password. Should I mention that here too? All the docker hub readmes will need the same instructions maybe confluence is a better place?